lib/wallaroo/core/initialization/application_distributor.pony

```diff
@@ -36,13 +36,16 @@ actor ApplicationDistributor is Distributor
   let _step_id_gen: StepIdGenerator = StepIdGenerator
   let _local_topology_initializer: LocalTopologyInitializer
   let _application: Application val
+  let _state_step_creator: StateStepCreator
 
   new create(auth: AmbientAuth, application: Application val,
-    local_topology_initializer: LocalTopologyInitializer)
+    local_topology_initializer: LocalTopologyInitializer,
+    state_step_creator: StateStepCreator)
```

Comment:
Remove StateStepCreator.  It's not used.

Reply:
done